core/templates/dev/head/pages/exploration_editor/editor_tab/UnresolvedAnswersOverviewDirective.js

```diff
@@ -26,71 +28,37 @@ oppia.directive('unresolvedAnswersOverview', [
         'unresolved_answers_overview_directive.html'),
       controller: [
         '$scope', 'EditorStateService', 'ExplorationStatesService',
-        'StateRulesStatsService', 'ExplorationRightsService',
+        'StateTopAnswersStatsService',
+        'SUPPORTED_HTML_RENDERINGS_FOR_INTERACTION_IDS',
         function(
             $scope, EditorStateService, ExplorationStatesService,
-            StateRulesStatsService, ExplorationRightsService) {
-          var MAXIMUM_UNRESOLVED_ANSWERS = 5;
-          var MINIMUM_UNRESOLVED_ANSWER_FREQUENCY = 2;
-
-          $scope.unresolvedAnswersData = null;
-          $scope.latestRefreshDate = new Date();
-          $scope.unresolvedAnswersOverviewIsShown = false;
-
-          $scope.computeUnresolvedAnswers = function() {
+            StateTopAnswersStatsService,
+            SUPPORTED_HTML_RENDERINGS_FOR_INTERACTION_IDS) {
+          /**
+           * Returns whether the answers for the current state are from an
+           * interaction ID which can be rendered as HTML.
+           *
+           * TODO(brianrodri): This would be more helpful/flexible in its own
+           * service, maybe a InteractionAnswerRenderingService?
```

Comment:
Technically, though, this is temporary. Perhaps the TODO should be: "Add all other interaction IDs to this list, then remove the list altogether."?